app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestStage.ts

```diff
@@ -0,0 +1,38 @@
+import { module } from 'angular';
+
+import {
+  PIPELINE_CONFIG_PROVIDER,
+  PipelineConfigProvider,
+  SETTINGS
+} from '@spinnaker/core';
+
+import { KubernetesV2DeployManifestConfigCtrl } from './deployManifestConfig.controller';
+
+import {
+  KUBERNETES_MANIFEST_COMMAND_BUILDER
+} from '../../../manifest/manifestCommandBuilder.service';
```

Comment:
I think this could be `'kubernetes/v2/manifest/..'`, which would read a litter nicer.

Reply:
I'm not sure how to make this one work
```
ERROR at /Users/Andrew/armory/deck/app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestConfig.controller.ts(6,8):
TS2307: Cannot find module 'kubernetes/v2/manifest/manifestCommandBuilder.service'.
```

I also tried:
```
ERROR at /Users/Andrew/armory/deck/app/scripts/modules/kubernetes/src/v2/pipelines/stages/deployManifest/deployManifestConfig.controller.ts(6,8):
TS2307: Cannot find module '@spinnaker/kubernetes/v2/manifest/manifestCommandBuilder.service'.
```